index.html

```diff
@@ -107,7 +107,7 @@ <h2>Tonic</h2>
                 </li>
             </ul>
             <div class="project-link-container">
-            <a href="#" class="project-link">See project</a>
+            <a id="projectButton1" href="#" class="project-link" data-see-project-button>See project</a>
```

Comment:
- [ ] Based on the project requirement, all the project cards' HTML code in your `index.html` should be removed and added dynamically using Javascript. The projects section should be created dynamically with the information from the projects array. **This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.** Check an example of students' projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/nicupop729/Portfolio/pull/11/commits/aabed37cc2208f52d0bd081a4cae8cdc83c49636
- [ ]  Modal popup data should be populated with the data from the projects array.

❗ ❗  Thanks to the previous 2 points, adding projects in the future will be easy. You will only add it in one place - as another object in the projects array.